evm/src/cpu/contextops.rs

```diff
@@ -163,6 +163,9 @@ fn eval_packed_set<P: PackedField>(
 
     // The next row's context is read from stack_top.
     yield_constr.constraint(filter * (stack_top[0] - nv.context));
+    for limb in &stack_top[1..] {
+        yield_constr.constraint(filter * *limb);
+    }
```

Comment:
nit: a tad clearer this way
```suggestion
    for &limb in &stack_top[1..] {
        yield_constr.constraint(filter * limb);
    }
```